### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by kshlm**
A new 'ocs-registry' target has been added to the Makefile. The other
targets have been updated to make the tools and the variables used
consistent.

The README has been update to use the Makefile and the consistent
variables.

### CODE REVIEW

**Comment by kshlm** [comment]
/hold

This depends on #19. Hold until it is merged.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Once the SDK is installed, the operator can be built via:
 ```console
 $ dep ensure --vendor-only
 
-$ operator-sdk build quay.io/openshift/ocs-operator
+$ make operator-sdk
```
Comment (obnoxxx) [inline README.md]: should this be `make ocs-operator` ?

Comment (kshlm) [reply README.md]: You're correct. I'll fix it.

This was just me trying to optimize how much I type. And now, I've typed so much more.

**Comment by kshlm** [comment]
@obnoxxx Is this review just for the Makefiles, or is it for #19 too? This PR is based on top of #19. I split it into 2 as these are for different topics.

**Comment by obnoxxx** [comment]
@kshlm, indeed let me review #19 first. :-)
I think my comment was for the "Update makefile" patch.

**Comment by obnoxxx** [comment]
#19 has been merged, @kshlm do you want to rebase this?

**Comment by kshlm** [comment]
@obnoxxx A rebase isn't so useful as the CI bot does merge commits any ways. After you give your approval, prow also ensures that it runs the tests once on a merged branch, before actually merging the change into master.

**Comment by obnoxxx** [comment]
> @obnoxxx A rebase isn't so useful as the CI bot does merge commits any ways.

Well ok. My brain is probably still too much wired to doing rebases. And the merge commit did only work smoothly because you updated this PR as well when you updated #19.... My approach would have been to update 19 first and get it merged, then update this. But it has worked nicely this way, too! :-) 

I would just try to avoid having merge commits that do conflict resolution.

I was also expecting two copies of the common patches, but apparently, looking at the current master branch, it has not happened, so all good! :-) (It would probably have happened if you hadn't rebased this on the exact commits that were then merged with #19.)

>  After you give your approval, prow also ensures that it runs the tests once on a merged branch, before actually merging the change into master.

**Pull request merged**